package.json

```diff
@@ -18,28 +18,29 @@
   ],
   "dependencies": {
     "@changesets/changelog-github": "^0.2.6",
-    "@changesets/cli": "^2.9.1",
-    "@frontity/core": "^1.7.1",
+    "@changesets/cli": "^2.9.2",
+    "@frontity/core": "^1.7.2",
     "@frontity/frontity-org-theme": "file:packages/frontity-org-theme",
-    "@frontity/google-analytics": "^1.1.0",
-    "@frontity/google-tag-manager": "^0.2.0",
+    "@frontity/google-analytics": "^1.2.0",
+    "@frontity/google-tag-manager": "^0.3.0",
```

Comment:
We are not using the `@frontity/google-tag-manager` package, are we? I guess you have to remove it from `package.json`.

Reply:
Yes, I removed it from the `frontity.settings.js` and forgot to delete it here.